schema/mappables/interIdentity.go

```diff
@@ -6,12 +6,17 @@
 package mappables
 
 import (
+	sdkTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/persistenceOne/persistenceSDK/schema/helpers"
 	"github.com/persistenceOne/persistenceSDK/schema/types"
 )
 
 type InterIdentity interface {
 	GetAuthentication() types.Property
 	GetExpiry() types.Property
-
```

Comment:
https://github.com/persistenceOne/persistenceSDK/pull/230/files#r640574415

This is it right?